### PR TITLE
Added link for contribution guidelines

### PR DESCRIPTION
**Pull request by ankingcodes**
Added a link for contribution guidelines in README because it contains information regarding building the project properly, which can help beginners, because README doesn't contain sufficient information.

### CODE REVIEW

**Comment by ankingcodes** [comment]
@wilzbach ping

**Comment by Geod24** [comment]
Doesn't Github show a message when you are first contributing, pointing to this guide ?

**Comment by ankingcodes** [comment]
Before contributing, I'd try to build the project properly. As I followed the instructions in the README, I first cloned, then got the DMD compiler and got everything set up. But there was no mention of `dub run` or `dub build` in the README. So, I had to look around and then found it was written in the Contributing.md. So, it would be good to have a link on the README.

File: README.md [hunk 1]
```diff
@@ -70,3 +70,4 @@ However, currently [2.076](https://dlang.org/changelog/2.076.0.html) is required
 # Contributing
 
 New contributers are always welcome, there's plenty to work on! For an easy start, take a look at issues marked [`bootcamp`](https://github.com/dlang/dub/labels/bootcamp)
+The contributing guidelines can be found [here](https://github.com/ankingcodes/dub/blob/master/CONTRIBUTING.md)
```
Comment (Geod24) [inline README.md]:
```suggestion
The contributing guidelines can be found [here](https://github.com/dlang/dub/blob/master/CONTRIBUTING.md)
```

**Pull request merged**